stablehlo/reference/Ops.cpp

```diff
@@ -179,6 +180,19 @@ Tensor evalReverseOp(const Tensor &operand, ArrayRef<int64_t> dimensions,
   return result;
 }
 
+Tensor evalSelectOp(const Tensor &pred, const Tensor &on_true,
+                    const Tensor &on_false, Type resultType) {
+  Tensor result(resultType);
+  for (auto it = result.index_begin(); it != result.index_end(); ++it) {
+    Element pred_value = pred.getType().getRank() != 0
```

Comment:
Can you extract the boolean value here to avoid importing `Element.h`?

Reply:
Oh the "Element.h" is actually already imported. Removed my import. The function is not changed: current implementation is more clear than combining the lines.

Reply:
Either is fine. One other comment, can you rename the variable to something like `predVal` to make it camel case?

Reply:
Sure!